admin/fleetcommander/utils.py

```diff
@@ -71,16 +82,22 @@ def config_to_dict(config):
         return res
 
     config = config_to_dict(config)
-    args['host'] = config[SECTION_NAME].get('host', args['host'])
-    args['port'] = config[SECTION_NAME].get('port', args['port'])
+    if host is None:
```

Comment:
What's the point of doing all this if we can put these values in the 'default' field of the argparser?